main/Main.hs

```diff
@@ -213,21 +227,33 @@ main = do
   -- ask for the OS time at most once per second
   getTime <- mkAutoUpdate defaultUpdateSettings {updateAction = getCurrentTime}
 
-  -- run the postgrest application
-  runSettings appSettings $
-    postgrest
-      conf
-      refDbStructure
-      pool
-      getTime
-      (connectionWorker
-         mainTid
-         pool
-         (configSchema conf)
-         refDbStructure
-         refIsWorkerOn)
+  let postgrestApplication = postgrest
```

Comment:
nitpick: To reduce indentation could you move the variable assignment to the next line? Like:
```haskell
let postgrestApplication =
  postgrest
    conf
    ...
```